docs/installation/upgrade-300.rst

```diff
@@ -67,3 +67,21 @@ Removal of legacy translations conversion
 Old (from before Open Forms 2.4) form export files containing form field translations
 in the legacy format are now ignored instead of converted to the new format. We
 recommend re-creating the exports on a newer version of Open Forms.
+
+Removal of single registration conversion
+-----------------------------------------
+
+The legacy format (from before Open Forms 2.3) for registration backend will no longer be
+converted to the current standard. When importing a form with this configuration,
+the form will be created without registration backends.
+
+Removal of single registration on forms
+=======================================
+
+The old registration backend configuration (from before Open Forms 2.3) is now completely
+removed. This removes `registrationBackend` and `registrationBackendOptions` from all api
+endpoints. You should be using the `registrationBackends` property.
```

Comment:
No need to document this - the API endpoints were labeled private API a while ago. The external world shouldn't have been using this in the first place :)

Reply:
I've removed this part of upgrade note :)